### PR TITLE
Addressing issue #139

### PR DESCRIPTION
**Pull request by aiwangmicrosoft**
https://github.com/Azure-Samples/active-directory-xamarin-native-v2/issues/139

### CODE REVIEW

File: 2-With-broker/README.md [hunk 1]
```diff
@@ -343,13 +343,14 @@ Make sure the platform you configured is the same one you mark for build and dep
 
 Clean the solution, build and run it:
 
-- Click the sign-in with broker button at the bottom of the application screen. 
-- The Authenticator App will open, or ask you to install it from the App Store. 
+- Click the sign-in with broker button at the bottom of the application screen. Depending on whether or not there are conditional access policies applied to the user signing in, there will be two different experiences:
+- If there are conditional access policies applied to the user signing in, the Authenticator App will open, or ask you to install it from the App Store. 
 - In the Authenticator App, either select an already existing account, or add a new one.
```
Comment (jmprieur) [inline 2-With-broker/README.md:348]: You probably want to intend lines 348 to 352?

Comment (aiwangmicrosoft) [reply 2-With-broker/README.md:348]: I was having trouble getting the indent to show up, let me try again today.
